src/lambda/models/samLambdaRuntime.ts

```diff
@@ -76,15 +53,6 @@ export function getFamily(runtime: string | undefined): SamLambdaRuntimeFamily {
         case 'dotnetcore':
         case 'dotnet':
             return SamLambdaRuntimeFamily.DotNetCore
-        case 'go1.x':
-        case 'go':
-            return SamLambdaRuntimeFamily.Go
-        case 'java8':
-        case 'java':
-            return SamLambdaRuntimeFamily.Java
-        case 'ruby2.5':
-        case 'ruby':
-            return SamLambdaRuntimeFamily.Ruby
         default:
             throw new Error(`Unrecognized runtime: '${runtime}'`)
```

Comment:
I want to change this eventually to something like `Unsupported runtime` or `Unsupported or unrecognized runtime` since SAM does support the runtime. Not changing it here because there is a test in `develop` that depends on the text here: https://github.com/aws/aws-toolkit-vscode/blob/develop/src/test/lambda/utilities/getMainSourceFile.test.ts#L149

Reply:
I like this idea, and would submit for consideration an UnsupportedRuntime Error class, containing the property `runtime`.

Reply:
The more I'm thinking about this, the less I'm sure if it'd be useful. The only time the error would pop would be when the runtime is passed to this function (or something similar), which would mean that the caller should already know what runtime is being used.